### PR TITLE
Membership Model Task Force

### PR DESCRIPTION
**Pull request by kariljordan**
Created task force charter for membership model. This task force charter has been reviewed by the Management Team. Please approve this PR for public record.

### CODE REVIEW

File: 2024/membership-model/membership-model-tf-charter.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- What measures can prevent a transactional relationship with members while ensuring sustainability?
+- How can membership benefits be expanded and enhanced to reflect The Carpentries' values and mission better?
+- What pricing structure best suits the organisation's operational costs and financial goals while emphasizing equity?
+- Which membership model system, such as a credit-based system, optimally complements our existing team's technical infrastructure and workflows?
+
+# Deliverables
+__The following documents and other outputs from this work will include:__  
+- [ ] A Member organisation survey aimed to identify the current needs of Carpentries member organizations. Success metrics include:  
+   -  [ ]Goal response rate: XX%
+   -  [ ]Goal # Member organization responses outside US/UK: #
```
Comment (tobyhodges) [inline 2024/membership-model/membership-model-tf-charter.md:26]:
```suggestion
   -  [ ] Goal response rate: XX%
   -  [ ] Goal # Member organization responses outside US/UK: # 
```

File: 2024/membership-model/membership-model-tf-charter.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- [ ]keeping the task force documentation up to date   
+- [ ]producing the final deliverables
+  
+__Board of Directors Representative, responsible for:__  
+- [ ]Attending the task force meetings, participating in discussions and decisions of the task force, and contributing to the tasks providing governance insight
+- [ ]Member Organisation Representative responsible for:
+- [ ]Attending the task force meetings, participating in discussions and decisions of the task force, and contributing to the tasks providing Member insight
+
+__Task Force Member, responsible for:__
+- [ ]Attending the task force meetings, participating in discussions and decisions of the task force, and contributing to the tasks
```
Comment (tobyhodges) [inline 2024/membership-model/membership-model-tf-charter.md:48]:
```suggestion
- [ ] overall leadership of the task force and advancing its aims and objectives  
- [ ] organising and chairing the task force meetings (e.g., preparing meeting agendas and details and distributing them ahead of meetings)  
- [ ] keeping the task force documentation up to date   
- [ ] producing the final deliverables
  
__Board of Directors Representative, responsible for:__  
- [ ] Attending the task force meetings, participating in discussions and decisions of the task force, and contributing to the tasks providing governance insight

__Member Organisation Representative responsible for:__
- [ ] Attending the task force meetings, participating in discussions and decisions of the task force, and contributing to the tasks providing Member insight

__Task Force Member, responsible for:__
- [ ] Attending the task force meetings, participating in discussions and decisions of the task force, and contributing to the tasks
```

File: 2024/membership-model/membership-model-tf-charter.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Task Force Charter: Exploring The Carpentries Membership Model
+### Approval date:
+### Status:
+### Completion date (planned): 
+
+# Problem Statement
```
Comment (tobyhodges) [inline 2024/membership-model/membership-model-tf-charter.md:6]:
```suggestion
## Problem Statement
```

File: 2024/membership-model/membership-model-tf-charter.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+Moreover, Members have expressed evolving needs and desires, seeking resources to enhance community sustainability, inclusion of curriculum development training, and support for workshop attendees post-training. Additionally, there's a growing interest in initiatives that foster inclusivity, such as recruiting and supporting underrepresented groups, facilitating cross-cultural collaborations, and engaging under-resourced communities.
+
+In response to these findings and ongoing discussions at the 2024 Board of Directors retreat, there's a pressing need to reassess The Carpentries' membership model. Key questions have been raised regarding aligning services with members' needs, defining success, and balancing revenue generation and community engagement. Moreover, recommendations have been proposed to introduce Collaborative Lesson Development  Training, ensure a non-transactional model, expand benefits, and align Membership offerings with The Carpentries' core values.
+
+# Objective
```
Comment (tobyhodges) [inline 2024/membership-model/membership-model-tf-charter.md:13]:
```suggestion
## Objective
```

File: 2024/membership-model/membership-model-tf-charter.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+This task force aims to revamp The Carpentries' Membership model by convening stakeholders to explore the following questions:
+- Are the current Membership offerings aligned with the evolving needs of Carpentries Member organizations?
+- How can success be defined in the context of the Carpentries Membership model, balancing revenue generation and community engagement?
+- What measures can prevent a transactional relationship with members while ensuring sustainability?
+- How can membership benefits be expanded and enhanced to reflect The Carpentries' values and mission better?
+- What pricing structure best suits the organisation's operational costs and financial goals while emphasizing equity?
+- Which membership model system, such as a credit-based system, optimally complements our existing team's technical infrastructure and workflows?
+
+# Deliverables
```
Comment (tobyhodges) [inline 2024/membership-model/membership-model-tf-charter.md:22]:
```suggestion
## Deliverables
```

File: 2024/membership-model/membership-model-tf-charter.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+- [ ] A Member organisation survey aimed to identify the current needs of Carpentries member organizations. Success metrics include:  
+   -  [ ]Goal response rate: XX%
+   -  [ ]Goal # Member organization responses outside US/UK: # 
+- [ ] Analysis of Member organisation survey data.
+- [ ] A recommendation for a new Membership model.
+- [ ] A project plan for piloting the revamped Membership model.
+- [ ] A blog post describing the conclusions reached by the task force
+- [ ] A document describing a communications plan for informing relevant community members (maintainers, contributors, lesson developers, CAC members) about the revamped Membership model.
+
+# Roles and Responsibilities
```
Comment (tobyhodges) [inline 2024/membership-model/membership-model-tf-charter.md:33]:
```suggestion
## Roles and Responsibilities
```

File: 2024/membership-model/membership-model-tf-charter.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+__Task Force Member, responsible for:__
+- [ ]Attending the task force meetings, participating in discussions and decisions of the task force, and contributing to the tasks
+
+# Operational Procedures
+- [Guiding Principles for Decision Making at The Carpentries](https://docs.google.com/document/d/13Ozf3jtzB95gFI5zVfMnhWXpExWtQkCZtog7h8FIwiQ/edit#heading=h.o6v3v3qszo7t)
+
+# Meetings
+Details of task force meetings are provided in the [Membership Model Task Force Strategy](https://docs.google.com/document/d/1cE-KKa_yHlW7tQwatfKP4mbdpbMoijzwO0t93iWHXg8/edit#heading=h.e60towfw575p)
+
+# Members
```
Comment (tobyhodges) [inline 2024/membership-model/membership-model-tf-charter.md:56]:
```suggestion
## Operational Procedures
- [Guiding Principles for Decision Making at The Carpentries](https://docs.google.com/document/d/13Ozf3jtzB95gFI5zVfMnhWXpExWtQkCZtog7h8FIwiQ/edit#heading=h.o6v3v3qszo7t)

## Meetings
Details of task force meetings are provided in the [Membership Model Task Force Strategy](https://docs.google.com/document/d/1cE-KKa_yHlW7tQwatfKP4mbdpbMoijzwO0t93iWHXg8/edit#heading=h.e60towfw575p)

## Members
```

**Review by tobyhodges — APPROVED** [review]
Looks good to me. I have suggested changes only to polish formatting+accessibility.

File: 2024/membership-model/membership-model-tf-charter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kariljordan) [reply 2024/membership-model/membership-model-tf-charter.md:26]: Yes, I will add them after the second task force meeting in August.

File: 2024/membership-model/membership-model-tf-charter.md (hunk 2 above)
Comment (kariljordan) [reply 2024/membership-model/membership-model-tf-charter.md:48]: Thank you SO MUCH for helping with formatting. My markdown skills were failing me today :)

**Review by froggleston — APPROVED** [review]
LGTM! 👍

**Review by ErinBecker — APPROVED** [review]

**Review by sheraaronhurt — APPROVED** [review]

**Pull request merged**